src/ngraph/frontend/onnx_import/utils/convpool.hpp

```diff
@@ -94,7 +94,37 @@ namespace ngraph
             {
                 return get_pads(node, get_kernel_shape(node));
             }
-        } // namespace attribute
+
+            /**
+             * @brief Create an nGraph pooling operation based on an ONNX pooling op.
+             *
+             * @tparam T Class of an nGraph pooling operation (e.g. AveragePool, MaxPool)
+             * @param node incoming ONNX opearation
+             * @return nGraph node equivalent of the ONNX operation
+             */
+            template <class T>
+            inline NodeVector make_ng_pool(const Node& node)
+            {
+                // Fetch input node for the pooling operation
+                auto data{node.get_ng_inputs().at(0)};
```

Comment:
`const auto& data = node.get_ng_input().at(0);`

Reply:
This returns `shared_ptr`, so using `&` doesn't work.